### ISSUE TITLE
Mission 2-3 Bug

### ISSUE THREAD

**Issue by Suff121** [EdenServer/community#1304]
Upon completing bastok side of mission, mission log bug and did not change therefore not allowing me to continue onto the windurst half of the mission.

**Comment by Shoruto** [comment]
Did you receive the message about needing to head to Windurst from the Sandorian Consulate? When you went to talk with them once you finished up the Bastok part?  It should of been the last thing they said.

**Comment by theTaikun** [comment]
I'm having a problem with Sandy 2-3 as well. I went to Bastok first, and got the CS at the consulate. Went to talk to Pius, but didn't get a CS. Went to Grohm and didn't get a CS or the pickaxes he's supposed to give.

I bought some pickaxes and got the mythril sand. Tried trading to Grohm, embassador, and Pius. Can't finish Bastok portion.

[Walkthrough](https://ffxiclopedia.fandom.com/wiki/Journey_Abroad#Bastok_Windurst_walkthrough)
